README.md

```diff
@@ -29,7 +29,7 @@ npm install moq-player
 Or include via a `<script>` tag (for the IIFE build):
 
 ```html
-<script src="https://cdn.jsdelivr.net/npm/package@latest/dist/moq-player.iife.js"></script>
+<script src="https://cdn.jsdelivr.net/npm/@moq-js/player@latest/dist/moq-player.iife.js"></script>
```

Comment:
Later we might want to pin to specific versions, but I think using latest is a good place to start. Thanks for updating this here!